src/main/java/com/brightpath/learnify/controller/mapper/DtoMapper.java

```diff
@@ -124,6 +129,23 @@ public QuizSimpleResult asQuizSimpleResult(QuizResultUpdateDto quizResultUpdateD
         return new QuizSimpleResult(quizResultUpdateDto.getIncorrect(), quizResultUpdateDto.getCorrect());
     }
 
+    public CommentCreation asCommentCreation(CommentCreationDto commentCreationDto, ResourceType resourceType, UUID resourceId) {
+        return new CommentCreation(commentCreationDto.getOwnerId(),
+                resourceType,
+                resourceId,
+                commentCreationDto.getRating().shortValue(),
+                Optional.ofNullable(commentCreationDto.getTitle()),
+                Optional.ofNullable(commentCreationDto.getDescription()));
+    }
+
+    public CommentDto asCommentDto(Comment comment) {
+        return new CommentDto(comment.id(),
+                comment.commentOwnerId(),
+                (int) comment.rating(),
```

Comment:
`int`?

Reply:
we cannot excplicty declare short in openApi. I used something like short, but its still Integer object